main.js

```diff
@@ -178,9 +178,39 @@ function logActivity() {
      }
  }
    saveCard();
+   clearTimerSection();
   }
 
-
+  function clearTimerSection() {
+    containerTimer.innerHTML = `<button class="create-new-activity">CREATE A NEW ACTIVITY</button>`;
+    mainTitle.innerText = 'Completed Activity';
+    var createNewActivity = document.querySelector('.create-new-activity');
+    createNewActivity.addEventListener('click', goHome);
```

Comment:
Should we be utilizing event delegation for this?

Reply:
For sure, let me know what you think of my updates!